# HG changeset patch
# User Doug Stewart <doug.dastew@gmail.com
# Date 1445721624 14400
#      Sat Oct 24 17:20:24 2015 -0400
# Branch stable
# Node ID 77b104cad58e887ba95f4176989803bcab25b3d4
# Parent  d29614aed16f4f65313743a46b982ea61ffe85e1
intro.txi change a to A to match the previous comments

diff --git a/doc/interpreter/intro.txi b/doc/interpreter/intro.txi
--- a/doc/interpreter/intro.txi
+++ b/doc/interpreter/intro.txi
@@ -189,17 +189,17 @@ use the left division operator, @samp{\}
 @end example
 
 @noindent
 This is conceptually equivalent to
 @tex
 $@code{A}^{-1}@code{b}$,
 @end tex
 @ifnottex
-@code{inv (a) * b},
+@code{inv (A) * b},
 @end ifnottex
 but avoids computing the inverse of a matrix directly.
 
 If the coefficient matrix is singular, Octave will print a warning
 message and compute a minimum norm solution.
 
 A simple example comes from chemistry and the need to obtain balanced
 chemical equations.  Consider the burning of hydrogen and oxygen to
